catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/suggestion/LatLon.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.query.suggestion;
+
+class LatLon {
```

Comment:
Do we really need this class? Why can't we use GeoTools coordinate or something?

Reply:
We need something that maps cleanly to JSON so the UI can transform it by name. It also has to be _stable_. I'd prefer not depending on a third party lib for this, or even a pojo that's outside the scope of `catalog-ui-search`. Just so a seemingly innocent change elsewhere doesn't introduce a bug.